task_spec/spec/acceptance/init_spec.rb

```diff
@@ -52,6 +52,10 @@ def add_node(node, group_name, groups)
       else
         # This is very unlikely to work
         node_name, node = host.hostname
+        node = {
```

Comment:
Will make this more dynamic as we develop the test further (example the hard coded "Administrator" user...). For now focusing on adding logic to BOLT-702.

Reply:
I think just wrapping it in conditional that checks if the host is windows. We don't want to test ssh against windows for now and beaker doesn't handle winrm connections(for the most part).

Is there an existing PASSWORD environment variable that beaker uses we can reuse here?